Common/NearestNeighbour/src/test/java/org/tribuo/common/nearest/TestKNN.java

```diff
@@ -184,4 +188,32 @@ public void knnRegressionInvalidExampleTest() {
         });
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void deserializeKNNRegressionV42ModelTest() {
+        String serializedModelPath = "src/test/resources/KNNTrainerRegressor_k3_L2_nt2_voting_streams_v4.2.model";
```

Comment:
Can you look it up as a classpath resource rather than use an explicit path into the source tree? That makes assumptions about what the working dir is when the tests are executed, and can cause problems when running them in an IDE.

Reply:
Sure. I occasionally run single tests in Intellij IDEA and didn't face an issue, but this suggestion should provide additional safety.